### ISSUE TITLE
Useful information about Vulkan extensions can no longer be discovered.

### ISSUE THREAD

**Issue by NicolBolas** [SaschaWillems/VulkanCapsViewer#64]
Relatively recent updates to the Vulkan Hardware Database's website make the system decidedly less useful.

I find that one of the most useful questions to answer is "what kind of hardware supports extension X"? I could find this out easily enough before by selecting X, then selecting supported, and checking which vendors are in the list.

That doesn't exist anymore. Instead, I get a table that tells me what percentage of reports on a platform support the extension. It's not even what percentage of distinct pieces of hardware; it's the report percentage, which is utterly useless unless it's 100% or 0%.

I understand the thinking behind the table approach; you're trying to condense the information down to something manageable. You're basically trying to answer "is this Android-specific, Android-neutral, or not available on Android?" And while that's a useful question, I think getting the Vendor string involved is *far* more useful. Does AMD support the extension; if so, on which hardware?

I wouldn't mind the current table breakdown if there was still some way to get the full listing.

It would also be great if there was some way to have these systems ignore old driver versions that expose the same hardware. That they should only look at the most recent hardware.

**Comment by SaschaWillems** [comment]
I have recently reworked how the extension listing page works. The percentage shown (per platform) does actually represent percentage of devices supported, and not (like in the beginning) percentage of reports.

Clicking on the percentage bar will also show you a list of devices supporting that extension for the given platform, e.g. like this: https://vulkan.gpuinfo.org/listdevices.php?platform=windows&extension=VK_AMD_draw_indirect_count

As for the missing vendor string: Adding that to the new device based table should be easy, so you could again filter the list of devices supporting a given extension by vendors. I'll try to add this as soon as I get some spare time.

I know that data representation is not where it should be, and if I was to redo the database I'd do things a bit different to make it easier to filter and aggregate relevant data, and the recent change of making the extension listing page device (percentage) based instead of report based was a first step that I hoped would make the database more useful.

**Comment by NicolBolas** [comment]
> Clicking on the percentage bar will also show you a list of devices supporting that extension for the given platform

It does work like a link, but it's not a regular link. If you mouse-over it, the mouse doesn't change. You also can't middle-click on it so that it pops up in a tab rather than taking over the current tab.

Also, it would be useful to see what hardware *doesn't* support the extension (the way we can with features). 

Also, it'd be good if you could get to those pages from the page for that specific extension. I find it useful to be able to link people to the extension page so that they can then see whether it is available on their hardware of choice. And as it currently stands, getting there requires some effort on the user's part.

**Comment by SaschaWillems** [comment]
You're right. Those are some valid points. I'll try to fix that and also add a way of displaying devices not supporting a given extension.
